code_generator/End2End/Tests/BlittableComponentTests.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Generated.Improbable.TestSchema;
+using Generated.Improbable.TestSchema.Blittable;
+using Improbable.Gdk.Core;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using Unity.Mathematics;
+
+namespace Improbable.Gdk.CodeGenerator.End2EndTests
+{
+    [TestFixture]
+    public class BlittableComponentTests
+    {
+        private static readonly bool1 Bool1False = new bool1(false);
+        private static readonly bool1 Bool1True = new bool1(true);
+
+        private const int IntValue = 123;
+        private const long LongValue = 5678L;
+        private const float FloatValue = 1.2345f;
+        private const double DoubleValue = 3.14159;
+        private static readonly bool1 BoolValue = Bool1True;
+
+        [Test]
+        public void component_should_be_an_ISpatialComponentData()
+        {
+            var component = new SpatialOSBlittableComponent();
+            Assert.True(component is ISpatialComponentData,
```

Comment:
Also assert that component is a `IComponentData`

Reply:
Done